src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +25,13 @@
      * In any other case throw DateTimeException.
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        switch (datePart) {
+            case FULL: return LocalDate.now().toString();
+            case YEAR: return String.valueOf(LocalDate.now().getYear());
+            case MONTH: return String.valueOf(LocalDate.now().getMonth());
+            case DAY: return String.valueOf(LocalDate.now().getDayOfMonth());
+            default: throw new DateTimeException("Incorrect input");
```

Comment:
I mightn't be correct, but according to Google java style each return statement has to be on the next line despite it's fitting after the case statement. I mean it's up to you

Reply:
done